airbyte-integrations/connectors/destination-postgres/src/main/resources/spec.json

```diff
@@ -52,13 +52,20 @@
         "airbyte_secret": true,
         "order": 5
       },
+      "ssl": {
+        "title": "SSL Connection",
+        "description": "Encrypt data using SSL.",
```

Comment:
`Connect using SSL`

Reply:
This should probably be the title actually, not sure what the description should be.

Reply:
I updated the description to

> Encrypt client/server communications for increased security.